packages/web/src/common/store/cache/collections/sagas.js

```diff
@@ -234,6 +235,9 @@ function* confirmCreatePlaylist(uid, userId, formFields, source) {
           })
         )
 
+        // Write out the new playlist to the playlist library
+        yield call(addPlaylistsNotInLibrary)
```

Comment:
just want to understand flow between this call, and the upload sagas one. will they not get called back-to-back?

Reply:
you would think so, and I did as well - however, it looks like the upload sagas does the same work create playlist does (it calls createPlaylist, etc), so we need to treat them distinctly